src/components/Reviews.js

```diff
@@ -1,28 +1,12 @@
 import React from 'react'
-import styled from 'styled-components/macro'
-import reviews from '../mocks/reviews.json'
+import Review from './Review'
 
-export default function Reviews() {
+export default function Reviews({ reviews }) {
   return (
     <>
       {reviews.map((review, index) => (
-        <Review key={index}>
-          <h2>{review.title}</h2>
-          <p>Rating: {review.rating}</p>
-          <h3>{review.categorie}</h3>
-          <h4>{review.subcategorie}</h4>
-          <h3>Summary: </h3>
-          <p>{review.summary}</p>
-          <h3>Lessons: </h3>
-          <p>{review.lessons}</p>
-        </Review>
+        <Review key={index} {...review} />
```

Comment:
Taking index as key is bad practice. When reviews data changes and its rendered again you can run into problems. Maybe you could take review.title?

Reply:
You are right I forgot about that one. review.title is a good Idea since this should be unique for each review.